### DISCUSSION TITLE
Unable to receive the PayPal bill for Copilot

### DISCUSSION THREAD

**Discussion by windziyu** [community/community#61212] [Copilot Conversations]
I tried Copilot for free before, and then cancelled the trial bill.Now I want to open a $10 per month plan，but I can't receive the PayPal bill.Why?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @windziyu , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
